cmd/common.go

```diff
@@ -256,36 +389,31 @@ func verifyBin(binPath []string) []string {
 	return w
 }
 
-func verifyKubeVersion(binPath []string) []string {
+func verifyKubeVersion(b string) []string {
 	// These executables might not be on the user's path.
 	// TODO! Check the version number using kubectl, which is more likely to be on the path.
 	var w []string
 
-	for _, b := range binPath {
-		_, err := exec.LookPath(b)
-		if err != nil {
-			w = append(w, fmt.Sprintf("%s: command not found on path - version check skipped\n", b))
```

Comment:
I think I will have to spend sometime looking at error handling as a whole. For now I will restore this message but only on asserting the verbose flag.